pkg/jobmanager/jobmanager.go

```diff
@@ -31,8 +31,8 @@ var log = logging.GetLogger("pkg/jobmanager")
 
 var cancellationTimeout = 60 * time.Second
 
-// errorPayload represents the payload carried by a failure event (e.g. JobStateFailed, JobStateCancelled, etc.)
-type errorPayload struct {
+// ErrorEventPayload represents the payload carried by a failure event (e.g. JobStateFailed, JobStateCancelled, etc.)
+type ErrorEventPayload struct {
 	Err string
```

Comment:
Personally I would preserve type error here and solve the JSON-marshaling problem using `MarshalJSON` method :)